### ISSUE TITLE
Fsleyes' requirements to easily work within shimming toolbox

### ISSUE THREAD

**Issue by po09i** [shimming-toolbox/shimming-toolbox#240]
### Summary
We have been having quite a bit of problems with the fsleyes installation so I wanted to clear things up once and for all (hopefully). 

### Fsleyes installation requirements depending on use case
In the plugin script, I removed all imports from shimmingtoolbox (only CLI calls remain). I was able to successfully launch the plugin whether fsleyes was opened from shimming toolbox's venv, the base conda venv or while not being in a venv at all. This means I got no errors while launching. Now here the path separates in two depending on which venv I was:

#### Other venv or no venv
The only thing not working were CLI calls since they can't be found as they are in st_venv. This can be fixed if we have launchers as proposed in issue #237

#### st_venv
Everything works 👏

### TLDR
The script or plugin needs to have no link (imports) to the project. The only references to the projects are the CLI calls. This allows to launch the plugin from venv/no venv. To use the CLIs:
- If we don't use launchers, then fsleyes need to be started from shimming toolboxes venv
- if we go with launchers, then fsleyes can be launched from anywhere

**Comment by po09i** [comment]
This can be closed as we introduced a new repo and packages it as a fsleyes plugin. The ST and fsleyes are therefore independent.
